src/verify/mod.rs

```diff
@@ -27,6 +27,17 @@ pub trait Credential {
     fn parse_claims(claims: ClaimsSet) -> Result<HashMap<String, ClaimValue>>;
 }
 
+pub fn decode_status_list(status_list: String, idx: usize) -> u8 {
```

Comment:
not blocking but the name feels a bit misleading to me since it's decoding + also retrieving a specific status list entry at `idx`, maybe something like `retrieve_entry_from_status_list` is clearer?